src/WebUI/wwwroot/RSG-25.json

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
I would prefer this file named more clearly, e.g. `translation.json` or `strings.json`.